spec/components/govuk_component/pagination_component_spec.rb

```diff
@@ -0,0 +1,255 @@
+require 'spec_helper'
+
+module Pagy::UrlHelpers
+  def request
+    OpenStruct.new(GET: {}, session: nil, host: "https://somesite/")
+  end
+end
```

Comment:
I suspect there's a nicer way of doing this, the usual 'pretend it's a request' tricks in RSpec weren't playing ball.

Reply:
Have you tried the [`with_request_url` test helper](https://viewcomponent.org/guide/testing.html#setting-requestpath_parameters)?

Reply:
Yeah, that was my first port of call. It wasn't being picked up from within Pagy.